docs/tasks/access-kubernetes-api/setup-extension-api-server.md

```diff
@@ -38,7 +38,7 @@ Alternatively, you can use an existing 3rd party solution, such as [apiserver-bu
 1. Create a Kubernetes service account in your namespace.
 1. Create a Kubernetes cluster role for the operations you want to allow on your resources.
 1. Create a Kubernetes cluster role binding from the default service account in your namespace to the cluster role you just created.
-1. Create a Kubernetes apiservice. The CA cert above should be base 64 encoded, stripped of new lines and used as the spec.caBundle in the apiservce.  This should not be namespaced.
+1. Create a Kubernetes apiservice. The CA cert above should be base 64 encoded, stripped of new lines and used as the spec.caBundle in the apiservice. This should not be namespaced. If using the [kube-aggregator API](https://github.com/kubernetes/kube-aggregator/blob/f44a75f488b92e99f8cd13640ef03a6aeda7be48/pkg/apis/apiregistration/types.go#L55-L56), only pass in the PEM encoded CA bundle because the base 64 encoding is done for you.
```

Comment:
I'm a little bit wary of the direct referencing of source code lines. It would be desirable to avoid this by writing some plain English text here.

Reply:
Updated to remove the direct link to source code lines and instead linked to the main `kube-aggregator` repo.